_pages/en_US/installing-boot9strap-(mset9).txt

```diff
@@ -14,7 +14,7 @@ title: "Installing boot9strap (MSET9)"
 
 ### Compatibility Notes
 
-This method requires a computer running Windows, Linux, or macOS. Phones or tablets are NOT currently supported. If you do not have access to a computer, you will need to use an [alternate exploit](alternate-exploits).
+This method requires a computer running Windows, Linux, or macOS. ChromeOS and Android devices have an app that is currently not covered here. iOS devices are not supported. If you do not have access to a computer, you will need to use an [alternate exploit](https://wiki.hacks.guide/wiki/3DS:Alternate_Exploits).
```

Comment:
do we want to have this as [an app](link) in the meantime?

Reply:
How would you word it?

"An Android application exists for this process; for further support, ask for help at [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)." ?

Reply:
We should handle this in #2402